lib/sales_engine.rb

```diff
@@ -13,7 +13,9 @@ class SalesEngine
               :invoice_items,
               :customers,
               :transactions,
-              :merchant_data
+              :merchant_data,
+              :transaction_data,
+              :invoice_item_data
 
```

Comment:
can already hear them now.... see if we can move these out.

Reply:
I could make manual attr_readers for them